packages/kbn-test/src/es/test_es_cluster.ts

```diff
@@ -248,6 +255,7 @@ export function createTestEsCluster<
           kill: true, // likely don't need this but avoids any issues where the ESS cluster wasn't cleaned up
           waitForReady: true,
         });
+        return;
```

Comment:
Good catch!

Reply:
it doesn't really make any difference, I just wanted to avoid misunderstanding